backend/server.py

```
@@ -120,7 +120,27 @@ def clear_error_log_buffer() -> None:
 
-SCHEDULER = AgentScheduler()
+def _scheduler_on_fire(job) -> None:
+    """Called by APScheduler when a cron fires. Dispatches to the orchestrator."""
+    import asyncio, threading
+    from services.workflow_orchestrator import get_workflow_orchestrator
+    async def _dispatch():
+        try:
+            orch = get_workflow_orchestrator()
+            run = await orch.execute(request=job.instruction, auto_approve=True)
+            log.info("Scheduler fired job %s → run %s", job.job_id, run.get("run_id"))
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)